package.json

```diff
@@ -68,8 +68,6 @@
     "copy-to-clipboard": "^3.0.8",
     "country-flag-icons": "^1.4.19",
     "currency-formatter": "^1.4.2",
```

Comment:
Same for this

Reply:
It's used in tnc-leg. It seems like we might not be using tnc-leg anymore though. Can it be removed?